pdoc/templates/default/module.html.jinja2

```diff
@@ -31,6 +70,21 @@
             }
 
             {% block style_layout %}
+                /* Colors of overall document */
+                body {
+                      --main-bg-color: #fff;
+                      --main-text-color: #212529;
+                      color: var(--main-text-color);
+                      background-color: var(--main-bg-color);
+                }
+
+                 @media (prefers-color-scheme: dark) {
+                      body {
+                          --main-bg-color: #212529;
+                          --main-text-color: #fff;
+                      }
+                 }
```

Comment:
Let's move the body colors specifications to the rest of the colors specs and can combine the two `@media (prefers-color-scheme: dark)` queries.

Reply:
Done in ceb59b0